### DISCUSSION TITLE
Don't eat blank lines in messages

### DISCUSSION THREAD

**Discussion by gdude2002** [stoatchat/discussions#89] [Feature Suggestions]
Note from maintainer: please see issue https://github.com/revoltchat/revite/issues/214.
---

This is a simple one - if you have a blank line in your message, Revolt removes it in the output.

Message:

```markdown
**Line one**

**Line three**
```

Output:

**Line one**
**Line three**

This means that it's functionally impossible to have any sort of blank line in your message.

**Comment by Yu-Vitaqua-fer-Chronos** [comment]
Yeah this is annoying, especially when sending paragraphs, because adding a space in between the paragraphs helps to space it out

**Comment by YTG1234** [comment]
> This means that it's functionally impossible to have any sort of blank line in your message.

It's not impossible, as you can still use a backslash:
```markdown
**Line one**
\
**Line three**
```

however, it is very annoying.

**Comment by insertish** [comment]
![image](https://user-images.githubusercontent.com/38285861/140333566-e0f17e99-21d6-485a-af53-2b50e54f2c48.png)

Fixed the issue, will be deployed to nightly soon, if there are any issues please comment on https://github.com/revoltchat/revite/issues/214.
